mitodl/ocw-studio
# add UI for editing website collection entries

#### Pre-Flight checklist



#### What are the relevant tickets?

closes #376
closes #397

#### What's this PR do?

This adds UI to the website collection editing interface which allows you to add websites to a website collection and also to re-order existing websites in the collection.

#### How should this be manually tested?

Go to `/collections` and check out a collection you have. There should be a new section in the edit form with a field for picking a website. If you pick one and click 'Add' a request should go off to the backend, and then it should appear in your list. Once you've added a few websites to your list you should be able to re-order them as you please.


#### Screenshots (if appropriate)

<img width="577" alt="Screen Shot 2021-07-15 at 2 59 01 PM" src="https://user-images.githubusercontent.com/6207644/125842621-050c6648-307f-494c-b62c-92a84255af88.png">
<img width="679" alt="Screen Shot 2021-07-15 at 2 59 25 PM" src="https://user-images.githubusercontent.com/6207644/125842623-a2049377-43a1-4999-8800-2e1b61ebf439.png">
<img width="628" alt="Screen Shot 2021-07-15 at 2 59 20 PM" src="https://user-images.githubusercontent.com/6207644/125842625-21857fe7-b9f2-41cb-b161-e3a78558cd3c.png">
<img width="657" alt="Screen Shot 2021-07-15 at 2 59 06 PM" src="https://user-images.githubusercontent.com/6207644/125842627-b7dc3e70-cef9-4eba-8fe0-13dfb938842c.png">
